modules/spock/src/main/groovy/org/testcontainers/spock/TestcontainersMethodInterceptor.groovy

```diff
@@ -50,6 +55,15 @@ class TestcontainersMethodInterceptor extends AbstractMethodInterceptor {
 
     @Override
     void interceptCleanupMethod(IMethodInvocation invocation) throws Throwable {
+        findAllBrowserWebDriverContainers(invocation).each {
```

Comment:
I was thinking about this and it seems that it will result in exporting the video after each failed test.. 
does it mean we have to restart the container after each test(failed or not)? 
or it will just stop the execution? 

maybe we want to do this at cleanupSpec time, not sure

Reply:
So we actually have to add it to both methods but have a conditional `findAllBrowserWebDriverContainers()` method, so we get the non-shared containers in `interceptCleanupMethod` and only the `@Shared` ones in `interceptCleanupSpecMethod`.